### ISSUE TITLE
Superuser request unable to be operated on Android11+ with a screen reader because the tapjacking protection toggle is not available anymore

### ISSUE THREAD

**Issue by CiprianDinca9** [topjohnwu/Magisk#7290]
Device:Samsung Galaxy S10+ on OneUI5.1 port ROM, but it happens to any device. This is a general bug
Android version:Since the tapjacking protection toggle isn't available in Magisk's settings
Magisk version name:I am using de00f1d5, but this happens since long ago. Very long ago
Magisk version code:26202

When using any screen readers in this case, the superuser request is unable to be operated. To us, it shows like a window with no elements. To work around this restriction, this screen reader can help, but you need to pay for it and use it's OCR. https://github.com/nirenr/jieshuo
But many don't have money to pay to use the OCR or don't want to. OCR needs internet and when there's none, then there's a problem as it won't work. The other problem with this workaround is that sometimes it won't click and you need to scan multiple times and the time might expire if their OCR engines are super used at the time or your internet is slow. The last thing is, Many people maybe wouldn't want to use Jieshuo. TalkBack is left completely out in the dark, Jieshuo free users are also left out in the dark.
Do you think you can do something so that the superuser request can be successfully operated with screen readers? Feel free to test with Talkback.
The way we were doing it is to set automatic response to grant. But that's totally insecure and I ran into some app that asked for superuser access to do stuff that's unlegit, so I finally found out how important the set to prompt is.
If you can and it's ok, please fix this!
Kind regards and many thanks! I'm looking further to discussing this with you if you need anything else.

**Comment by CiprianDinca9** [comment]
I have updated and am still updating builds on Canary. I have switched to 26023 as well. Let me know when the problem is worked around. Kind regards!

**Comment by simsimient1** [comment]
Canary, not fucker

**Comment by CiprianDinca9** [comment]
> Canary, not fucker

Did not I kindly say that the bug happens on any version since Android11 came out? Does it really matter if I'm using Canary or not? Is this bug related to the instability of Canary you think? Then take your stupid stable, get your screen reader on, set the automatic response to prompt and see what you can do.
And for your piece of mind, I'm using Canary to get the change earlier and to get a probable fix earlier. But what can you understand really? You didn't understand that this bug does have nothing to do with the version I'm running. You better say anything nice if you want to actually say something.

**Comment by yujincheng08** [comment]
I think this is a duplicate of #2125, which wants to improve talkback, and there's a WIP improvement #6725.

**Comment by CiprianDinca9** [comment]
I know about that issue, but that thing is not a problem anymore and this is a completely different issue.

**Comment by CiprianDinca9** [comment]
This needs nothing but the elements of the superuser dialog are visible, just like they are on Android 10 and lower with tapjacking protection off. Nothing much. The Magisk app is completely accessible now. This is the only thing that needs fixed as we need to set the auto response to prompt for now or use Jieshuo with OCR.
